### PR TITLE
Adding pages for initial login and signup to enter into VA

### PR DESCRIPTION
**Pull request by parisha6**
The initial pages for VA are created with details being stored in AWS db. Please review the PR request. 

Best Regards,
Parisha Desai

### CODE REVIEW

File: Sprint 1 PD/app.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app)
+
+# Configuration for the SQLite database
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://admin:pfwfall2024@donnadb.cvaai4e089sw.us-east-2.rds.amazonaws.com:3306/appdetail'
```
Comment (zeshengchen) [inline Sprint 1 PD/app.py:9]: put user name, password, and the link to db in the environment variables in ".env" file.

File: Sprint 1 PD/.env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DATABASE_URL=jdbc:mysql://donnadb.cvaai4e089sw.us-east-2.rds.amazonaws.com:3306
```
Comment (zeshengchen) [inline Sprint 1 PD/.env:1]: do not directly upload ".env" file, instead uploading ".env_example" file and put something like:
DATABASE_URL=
...
in ".env_example" file

**Review by usha8719 — APPROVED** [review]
Looks good

**Review by ashutoshm97 — APPROVED** [review]

**Review by swetha-r13 — APPROVED** [review]

**Comment by zeshengchen** [comment]
@parisha6 It looks like that you just totally ignored my peer review comments.

**Comment by parisha6** [comment]
Professor I saw your comments. I wanted to action on them in next PR so I have tested it on my local and keep it in working condition in local before doing changes in Github.

Is that ok? I will reply on individual conversation.

I may have not followed the professional standards please excuse me for same. I am learning the process.

Best Regards,

Parisha Desai
Graduate Student -MSCS
Purdue University Fort Wayne
Email: ***@***.******@***.***>

**Comment by zeshengchen** [comment]
One of reasons to use pull request is to get the feedback from peer reviews and then fix them before completing the pull request. For future pull requests, please fix all peer review comments before completing them. If you have any questions, please let me know.

**Comment by parisha6** [comment]
Ok Professor. Sure I will be mindful of same.

Thanks.

Best Regards,

Parisha Desai
Graduate Student -MSCS
Purdue University Fort Wayne
Email: ***@***.******@***.***>

**Pull request merged**